index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="ham-cross">
+                <span class="hamburger-icon1">&#9776</span>
+                    <span class="cross hidden">X</span>
+            </div>
+            <div class="logo-items">
+                <span class="nav-logo"> <a href="index.html"> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                    <ul class="second-navbar">
+                        <li class="second-navbars-links"><a href="about.html">About</a></li>
+                        <li class="second-navbars-links"><a href="#"> Products</a></li>
+                        <li class="second-navbars-links"><a href="#"> News</a></li>
+                        <li class="second-navbars-links"><a href="#"> login</a></li>
+                    </ul>
+                </div>
+        <div class="second-navbar-popup hidden">
+            <ul class="second-navbar-popup-items">
+                <li class="second-navbar-popup-links"><a href="about.html">About</a> </li>
+                <li class="second-navbar-popup-links"><a href="#"> Products</a></li>
+                        <li class="second-navbar-popup-links"><a href="#"> News</a></li>
+                        <li class="second-navbar-popup-links"><a href="#"> login</a></li>
+            </ul>
+        </div>
+        </nav>
+    </header>
```

Comment:
- [ ] The navbar is missing the top dark section with social media links, logout, and language links aligned to the right.
- [ ] The Login button should have a red border as seen in the design guideline below.

Design Guideline
![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/aa177574-df5f-4644-8abd-6404b8b605d1)

Your work
![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/b91dae44-8458-4e2d-9282-8aa3b6806148)